conf/responsiblePeople.routes

```diff
@@ -94,6 +94,9 @@ POST        /training/:index/                                      @controllers.
 GET         /fit-and-proper/:index/                                @controllers.responsiblepeople.FitAndProperController.get(index:Int, edit: Boolean ?= false, flow: Option[String] ?= None)
 POST        /fit-and-proper/:index/                                @controllers.responsiblepeople.FitAndProperController.post(index:Int, edit: Boolean ?= false, flow: Option[String] ?= None)
 
+GET         /approval-check/:index/                                @controllers.responsiblepeople.ApprovalCheckController.get(index:Int, edit: Boolean ?= false, flow: Option[String] ?= None)
```

Comment:
URL should be approval test

Reply:
Why?

Reply:
Checked with Martin. Should be check.

Reply:
Same as above ... the text and the info must be aligned.

Reply:
its been changed in the other branch